build-packages/webpack-i18n-runtime/build-config/webpack-i18n-import-loader/index.js

```diff
@@ -0,0 +1,95 @@
+const path = require('path');
+const fs = require('fs');
+const loaderUtils = require('loader-utils');
+
+const parentThemeHelper = require('@scandipwa/scandipwa-dev-utils/parent-theme');
+const extensions = require('@scandipwa/scandipwa-dev-utils/extensions');
+const getEnabledLocales = require('../../shared/getEnabledLocales');
+
+const childRoot = process.cwd();
+const parentRoots = parentThemeHelper.getParentThemePaths(childRoot);
+const extensionsRoots = extensions.map((extension) => extension.packagePath);
+
+function getFilePathsForLocale(locale) {
+    return [
+        childRoot,
+        ...parentRoots,
+        ...extensionsRoots
+    ].reduce(
+        (acc, root) => {
+            const potentialPath = path.join(root, 'i18n', `${locale}.json`);
+            if (fs.existsSync(potentialPath)) {
+                return acc.concat(potentialPath);
+            }
+
+            return acc;
+        },
+        []
+    );
+}
+
+function generateImportDeclaration(path, chunkName) {
+    return `import(/* webpackMode: "lazy", webpackChunkName: "${chunkName}" */ '${path}')`;
+}
+
+function generateLocaleMapContents(localePathMap, defaultLocale) {
+    return Object.entries(localePathMap).reduce(
+        (acc, [localeKey, localePaths = []]) => {
+            const importsForEntry = localePaths.map(
+                (localePath) => generateImportDeclaration(localePath, localeKey)
+            );
+
+            const importingFunction = `function() { return [\n${importsForEntry.join(',\n')}\n] }`;
```

Comment:
Why is the \n here? Seems like an unnecessary symbol in the final bundle.

Reply:
It will not be in the final bundle, the code that is produced here will be minified afterward. 
This is introduced for easier debugging.